### DISCUSSION TITLE
[잡담]파이썬 연습중 질문입니다...!

### DISCUSSION THREAD

**Discussion by NA-DK** [4Moyede/HexatonClass-01#65] [잡담]
간단한 코딩 혼자 연습중인데...질문거리가 생겨서 어디에 올려야 할지 몰라 여기에 올리겠습니다...!
for age in [22,21,17,32,4,28,19,8]:
...     if age>=20:
...         total_price = total_price + 8000;
...     elif 10<=age<20:
...         total_price = total_price + 5000;
...     else:
...         total_price = total_price + 2500;
...     print('total_price')
NameError: name 'total_price' is not defined

에러에 total_price가 정의되어 있지 않다고 나옵니다...

**Comment by yoomambo** [comment]
안녕하세요 덕규멘티님 @NA-DK 멘토 유혁준입니다.

우선 저번에 강의시간에 말씀드렸던대로 Issue에서

1. New issue
2. Python Code 클릭
3. 형식대로 적기 (code 입력할 때는 
\```python
 {content}
\```
이 마크다운 수식어를 꼭 써주셔서 입력해주세요.

Issue에 등록하시면 답변 드리겠습니다.

감사합니다.
